services/QuillLMS/lib/tasks/app_settings.rake

```diff
@@ -35,7 +35,7 @@ namespace :app_settings do
 
     app_setting = AppSetting.find_by_name!(args[:name])
 
-    app_setting.update!(user_ids_allow_list: app_setting.user_ids_allow_list.concat(user_ids))
+    app_setting.update!(user_ids_allow_list: app_setting.user_ids_allow_list.uniq.concat(user_ids))
```

Comment:
Does it make sense to do the `.uniq` after the `.concat` so that if the new user_ids include dupes, we still aren't saving duplicates?

Reply:
Good idea